src/Storage/Device/MinIO.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace Utopia\Storage\Device;
+
+use Utopia\Storage\Device\S3;
+
+class MinIO extends S3
+{
+    /**
+     * Regions constants
+     *
+     */
+    const EU_CENTRAL_1='eu-central-1';
+    const US_SOUTHEAST_1='us-southeast-1';
+    const US_EAST_1='us-east-1';
+    const AP_SOUTH_1='ap-south-1';
+
+    /**
+     * Object Storage Constructor
+     *
+     * @param string $root
+     * @param string $accessKey
+     * @param string $secretKey
+     * @param string $protocol
+     * @param string $host
+     * @param string $bucket
+     * @param string $region
+     * @param string $acl
+     */
+    public function __construct(string $root, string $accessKey, string $secretKey, string $protocol, string $host, string $bucket, string $region = self::EU_CENTRAL_1, string $acl = self::ACL_PRIVATE)
```

Comment:
```suggestion
    public function __construct(string $root, string $accessKey, string $secretKey, string $host, string $bucket, string $region = self::EU_CENTRAL_1, string $acl = self::ACL_PRIVATE)
```
https://github.com/minio/minio-js/blob/master/src/main/minio.js#L70
Please avoid changing the signature of the constructor. These days it is very uncommon to use HTTP so let's default  to HTTPS like the official SDK.